### PR TITLE
Call hexagon delegate only if TFLITE_ENABLE_HEXAGON is defined.

### PR DESCRIPTION
**Pull request by NAgarwalla**

### CODE REVIEW

File: tensorflow/lite/tools/evaluation/utils.cc [hunk 1]
```diff
@@ -135,6 +135,7 @@ TfLiteDelegatePtr CreateGPUDelegate() {
 #endif  // TFLITE_SUPPORTS_GPU_DELEGATE
 }
 
+#if defined(TFLITE_ENABLE_HEXAGON)
```
Comment (multiverse-tf) [inline tensorflow/lite/tools/evaluation/utils.cc:138]: I think this change will lead to some break for bazel-based builds where hexagon delegate is built based on platform-related macros? So, could you change cmake-related configurations (similar to the TFLITE_WITHOUT_XNNPACK macro) to achieve the same purpose?

just fyi, @terryheo @karimnosseir

Comment (NAgarwalla) [reply tensorflow/lite/tools/evaluation/utils.cc:138]: The build is actually broken currently when TFLITE_ENABLE_HEXAGON is **not** defined. I believe it is defined by default.

Comment (gbaned) [reply tensorflow/lite/tools/evaluation/utils.cc:138]: @multiverse-tf Can you please assist on above comments from @NAgarwalla. Thanks!

**Review by karimnosseir — CHANGES_REQUESTED** [review]
Can you retry from master branch and check if the build is still broken ?

Thanks

**Comment by NAgarwalla** [comment]
This was indeed fixed as part of https://github.com/tensorflow/tensorflow/commit/302d9b34926108103a54ac8ae6ad63c93e83302f so am closing this.  Thanks @karimnosseir for fixing this!

**Pull request closed without merging**